docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -45,6 +45,16 @@ function hs_layouts_preprocess_container(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function hs_layouts_preprocess_details(&$variables) {
```

Comment:
Please remove this preprocess, it's not doing anything as you're using JS for collapsing the element.

Reply:
Ah, yes, I was wondering. I saw this worked at first, but then realized that JS was overriding it on page load.